pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <artifactId>opensrp-server-connector</artifactId>
     <packaging>jar</packaging>
-    <version>2.2.5-SNAPSHOT</version>
+    <version>2.2.6-SNAPSHOT</version>
```

Comment:
Does this break backward compatibility? If so we can increment the other semantic digits

Reply:
yeah let me update